### PR TITLE
Add Qase icon

### LINKED ISSUE #9321: Icon Request: Qase.io
### Brand Name

Qase

### Website

https://qase.io/

### Popularity Metric

The Similarweb rank is 191,910. See: https://www.similarweb.com/website/qase.io/#overview

### Official Resources for Icon and Color

png: https://qase.io/images/q_logo.png
color: #44268F, from the logo

### Additional Comments

Qase is a modern test management platform for manual + automated QA testing, tracking, and reporting that helps deliver higher quality software, faster.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by lananolana**
**Issue:** closes #9321

**Similarweb rank:** 267,966. See [here](https://www.similarweb.com/website/qase.io/#overview).

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Qase

Qase is a modern test management platform for manual + automated QA testing, tracking, and reporting that helps deliver higher quality software, faster.

### CODE REVIEW

**Comment by PratyushJayachandran** [comment]
Hello @lananolana 
It looks like we will need thier permission before using the icon. Refer: https://qase.io/terms
Can you inquire them about permission? If granted, just post a sceeenshot here and then we will go from there.

> Intellectual property rights
This Agreement does not transfer to you any intellectual property owned by Qase Inc or third-parties, and all rights, titles, and interests in and to such property will remain (as between the parties) solely with Qase Inc. All trademarks, service marks, graphics and logos used in connection with our Website or Services, are trademarks or registered trademarks of Qase Inc or Qase Inc licensors. Other trademarks, service marks, graphics and logos used in connection with our Website or Services may be the trademarks of other third-parties. Your use of our Website and Services grants you no right or license to reproduce or otherwise use any Qase Inc or third-party trademarks.

**Comment by UmangLadha** [comment]
download

**Comment by UmangLadha** [comment]
> download

here is your logo sir

**Review by UmangLadha — APPROVED** [review]

**Comment by n3r** [comment]
Hello @PratyushJayachandran!

My name is Nikita and I'm Qase CEO and founder. I approve logo usage :)

P.S. @lananolana, thanks for adding!

**Comment by PratyushJayachandran** [comment]
> Hello @PratyushJayachandran!
> 
> My name is Nikita and I'm Qase CEO and founder. I approve logo usage :)
> 
> P.S. @lananolana, thanks for adding!

Thank you for the confirmation chief. Will get this merged 😊

File: icons/qase.svg
Comment (PratyushJayachandran) [inline icons/qase.svg:1]: @lananolana I see [some differences](https://github.com/simple-icons/simple-icons/assets/35000807/311260c5-ad11-4029-aa9c-5607270fc5cb) when laying the source image turned white over your contribution. Can you give it another go? Please be careful to not round off or let your editing application round off the dimensions when scaling and positioning the icon.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10837,6 +10837,11 @@
             "hex": "E40000",
             "source": "https://www.qantas.com"
         },
+        {
+            "title": "Qase",
+            "hex": "4F46DC",
+            "source": "https://qase.io"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:11144]:
```suggestion
            "source": "https://qase.io",
            "license": {
                "type": "custom",
                "url": "https://github.com/simple-icons/simple-icons/pull/9907#issuecomment-1831377594"
            }
```
Please add this to indicate special permission granted for use of Qase icon in simple-icons

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Please make the requested changes @lananolana 😄

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @lananolana?

**Comment by lananolana** [comment]
> [@lananolana ](/lananolana)
> ` [Qase](/simple-icons/simple-icons/pull/9907/commits/70d59fdfa0d963b5058fc9e0669b9eb41b7c11dc)`

Done!

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for the update, @lananolana, but I'm still seeing the same differences as @PratyushJayachandran above. It looks like this is simply a result of your version becoming vertically stretched - the "Q" in the source SVG (assuming you're using [this one](https://qase.io/images/qase-logo-white.svg) from the website's header isn't perfectly square, so resizing it to 24dp in width should result in a height of ~23.895dp.

**Comment by lananolana** [comment]
> Thanks for the update, @lananolana, but I'm still seeing the same differences.

I use [this one](https://app.qase.io/assets/main/images/logo-night.698f1aa4.svg), but nevertheless Q is the same. Please check out my new attempt to work with svg.

**Review by PeterShaggyNoble — APPROVED** [review]
That looks spot on now, @lananolana 👍🏻 Thanks for your work on this one.

**Pull request merged**